site/en/blog/watch-video-using-picture-in-picture/index.md

```diff
@@ -18,7 +18,7 @@ Picture-in-Picture (PiP) allows users to watch videos in a floating window
 (always on top of other windows) so they can keep an eye on what they’re
 watching while interacting with other sites, or applications.
 
-With the new [Picture-in-Picture Web API], you can initiate and control
+With the [Picture-in-Picture Web API], you can initiate and control
```

Comment:
```suggestion
With the [Picture-in-Picture Web API], you can start and control
```

Reply:
A best practice in English writing is to use simpler words when they're available.

Reply:
Thanks!